services/meta/errors.go

```diff
@@ -55,6 +55,10 @@ var (
 	// with an existing policy.
 	ErrRetentionPolicyConflict = errors.New("retention policy conflicts with an existing policy")
 
+	// ErrRetentionPolicyDurationBelowShardDuration is returned when updating a retention
+	// policy that has a duration lower than the current shard duration.
+	ErrRetentionPolicyDurationBelowShardDuration = errors.New("retention policy duration must be greater than the shard duration")
```

Comment:
I can do that. But, I'm going to do `ErrRetentionPolicy` as the prefix since that's the naming standard. I know it's long, but I don't want to break that naming. So is `ErrRetentionPolicyIncompatibleDurations` fine?